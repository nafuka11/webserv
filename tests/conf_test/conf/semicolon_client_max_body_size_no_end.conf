client_max_body_size 1000

server {
    server_name webserv01;
    listen 4242;
}