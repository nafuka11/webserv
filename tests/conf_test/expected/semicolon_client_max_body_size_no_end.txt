webserv: directive "client_max_body_size" has no end ";" in ./conf/semicolon_client_max_body_size_no_end.conf:1
